mmv1/third_party/terraform/tests/resource_compute_network_endpoint_test.go.erb

```diff
@@ -188,7 +188,7 @@ data "google_compute_image" "my_image" {
 // was destroyed properly.
 func testAccCheckComputeNetworkEndpointWithPortsDestroyed(t *testing.T, negId string, ports ...string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
-		foundPorts, err := testAccComputeNetworkEndpointsListEndpointPorts(t, negId)
+		foundPorts, err := testAccComputeNetworkEndpointListEndpointPorts(t, negId)
```

Comment:
Why was this previously working?

Reply:
This wasn't actually an error, it just didn't match the naming of the other functions here. I wanted to use this function name in `resource_compute_network_endpoints_test.go.erb`, so I needed to change it here first.